planning/behavior_velocity_traffic_light_module/src/scene.cpp

```diff
@@ -199,7 +199,9 @@ bool TrafficLightModule::modifyPathVelocity(PathWithLaneId * path, StopReason *
         input_path, lanelet_stop_lines,
         planner_param_.stop_margin + planner_data_->vehicle_info_.max_longitudinal_offset_m,
         planner_data_->stop_line_extend_length, stop_line_point, stop_line_point_idx)) {
-    RCLCPP_WARN_THROTTLE(logger_, *clock_, 5000, "Failed to calculate stop point and insert index");
+    RCLCPP_WARN_STREAM_ONCE(
+      logger_, "Failed to calculate stop point and insert index for regulatory element id "
+                 << traffic_light_reg_elem_.id());
```

Comment:
@shmpwk @tkimura4 @satoshi-ota are you alright with making this warn once? (Instead of every 5 seconds)

This is to reduce clutter.